to-do-list/src/API/PostService.tsx

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios'
+import {PostItemProps} from '../components/Interfaces/interfaces'
+
+export default class PostService {
+    static async getAll(){
+        try{
+            const response = await axios.get('db.json')
+            return response.data
+        } catch(e){
+            console.log(e)
+        }
+    }
+    static async create(post: PostItemProps) {
+        try {
+          const response = await axios.post('http://localhost:3006/tasks', post);
```

Comment:
It's better to extract http://localhost:3006 to constant and use it (or you can create a .env file and place this value there)